src/vnsw/opencontrail-vrouter-netns/opencontrail_vrouter_netns/vrouter_hyperv.py

```diff
@@ -52,12 +135,13 @@ class HyperVManager(object):
     def __init__(self, vm_uuid, nic_left, nic_right, wingw_vm_name=None,
```

Comment:
I don't like init with some many arguments. I think that you should design this in different way (for example use dict as argument) https://stackoverflow.com/questions/5899185/class-with-too-many-parameters-better-design-strategy